examples/lock-app/cc13x2x7_26x2x7/README.md

```diff
@@ -226,15 +226,15 @@ This is done by scanning a QR code. A URL will be displayed on the lock-app's
 log ([UART terminal](#viewing-logging-output)). It will look like the following:
 
 ```
-SetupQRCode:  [CH:.81TM -00 0C9SS0]
+SetupQRCode:  [MT:.81TM -00 0C9SS0]
```

Comment:
I expect the value here, and others, are wrong since we switched to base-39.  Followup issue to fix them?